sw/SoC/examples/blinky/Makefile

```diff
@@ -28,14 +30,16 @@ STARTUP = 	      startup
 # OBJDUMP = $(RV_PREFIX)objdump $(32BITODFLAGS)
 #
 
-RV_PREFIX = 	riscv32-unknown-elf-
```

Comment:
Same as https://github.com/MaistoV/UninaSoC/pull/14/files#r1861817786 , remove this and assume `riscv32-unknown-elf-` is in PATH